### PR TITLE
Blue Genetics Airlocks

### PR DESCRIPTION
**Pull request by 13spacemen**
## What this does

All genetics/cloning airlocks are now blue, also added to RCD

## Pics

Pics are outdated, only the genetics/cloning airlocks are blue, not all of medbay

Screenshot 2023-12-30 001319

## Why it's good

matches geneticist color scheme

## Changelog

:cl:
 * rscadd: Genetics/cloning airlocks are now blue

### CODE REVIEW

File: maps/LampreyStation.dmm [hunk 1]
```diff
@@ -80913,7 +80913,7 @@
 /turf/simulated/floor,
 /area/lamprey/miniatureboxstationreplica)
 "dvm" = (
-/obj/machinery/door/airlock/medical{
+/obj/machinery/door/airlock/virology
```
Comment (Kurfursten) [inline maps/LampreyStation.dmm:80916]: typo, missed {

**Comment by CrazyAmphibian** [comment]
the blue blends in a bit easily with windows and the floor tiles, i suggest darkening it a bit to get better contrast for faster readability.

**Comment by 13spacemen** [comment]
@CrazyAmphibian i can't sprite at all chief, the sprites are from paradise
the photos don't do it justice i think a testmerge and poll would be good

**Comment by DeityLink** [comment]
Blue is the color of genetics, and arguably paramedic. We've gone over this debate in the past, and there are actually more items in the game that reflect's Medbay and medical doctors' color as green rather than blue. Eventually we settled on the current status quo of accepting both blue and green as medbay colors. I'd rather we keep it as such rather than enforce the hegemony of a single color over medbay.

**Comment by 13spacemen** [comment]
@DeityLink i'm ready to change the green crosses to blue but i'd like to get this in first
i think the blue fits a hospital/medical theme better

**Comment by falcon2346** [comment]
Hate this. If it gets merged I'm repainting the doors every round.

**Comment by MercuriusGH** [comment]
if it's going to be this kind of blue, why isn't it the same as the tiles

**Comment by wotisjanitor** [comment]
looks like TG

**Comment by GreatBigFailure** [comment]
I think a darker, almost navy blue would be good. I don't like the current shade, it doesn't pop out enough

**Comment by 13spacemen** [comment]
> if it's going to be this kind of blue, why isn't it the same as the tiles

it is the same as the tiles

**Comment by MercuriusGH** [comment]
my colour picker says the airlock stripe is 5995ba
floor tile is largely 8bbbd5 and 8cbcd6
no wonder it looks uncanny

**Comment by Kurfursten** [comment]
Breaking ties is never easy, but I don't think we should rewrite an entire department's identity unless there's more consensus and a tailor-made sprite instead of using one that doesn't match our tiles.

I could possibly see better support for dark blue doors to paramedic station.

**Comment by 13spacemen** [comment]
Ok now only applies to genetics airlocks, not all of medbay

**Pull request closed without merging**